tox.ini

```diff
@@ -9,17 +9,7 @@ deps = keyring
        django
        webtest
        nose
+       coverage>=3.6,<3.99
        unittest2
 setenv = PYTHONPATH=../google_appengine
-
-[testenv:py26]
-commands = nosetests --ignore-files=test_oauth2client_appengine\.py
```

Comment:
It looks like the file got removed sometime between 2013 and now.  I'm trying to look through the git logs to find when it got removed.